app/controllers/events_controller.rb

```diff
@@ -0,0 +1,48 @@
+class EventsController < ApplicationController
+  def new
+    @event = Event.new
+    @user = current_user
+    @future_event = future_event?
+  end
+
+  def create
+    @event = current_user.events.build(event_params)
+    if @event.save
+      flash[:success] = "Event Saved!"
+      redirect_to events_path
+    else
+      flash[:warning] = "All field must be filled!"
+      render :root_url
+    end
+  end
+
+  def index
+    @events = Event.all
+    @past_events = Event.previous_events
+    @upcoming_events = Event.upcoming_events
+    @user = current_user
```

Comment:
Guys, you don't need to assign `current_user` to an instance variable, it is available in the views because you defined it in the helpers.